apps/XiaoMiTool/install

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+install_packages openjfx adb android-tools-fastboot openjdk-11-jdk || exit 1
+
+git_clone https://github.com/francescotescari/XiaoMiToolV2 -b linux || exit 1
+cd XiaoMiToolV2
+echo "Building XiaoMi Tool..."
+./gradlew build || error "Failed to build XiaoMiToolV2!"
```

Comment:
adding my comment from the issue:
there will be problems with this if the user already has openjdk-11 installed
its possible for java 8 or some other lower version to be the default java, which gradle will then use and will fail (as seen before when someone tried to build on java 8)

potential solution:
```
case "$arch" in
    "32") JAVA_HOME=/usr/lib/jvm/java-11-openjdk-armhf ./gradlew build || error "Failed to build XiaoMiToolV2!" ;;
    "64") JAVA_HOME=/usr/lib/jvm/java-11-openjdk-arm64 ./gradlew build || error "Failed to build XiaoMiToolV2!" ;;
    *) error "Failed to detect OS CPU architecture! Something is very wrong." ;;
esac
```